src/server/mautic/mauticAPI.js

```diff
@@ -43,24 +43,38 @@ export const Mautic = {
     if (!(code && user.fullName && user.mauticId && Config.mauticVerifyEmailId))
       throw new Error('missing input for sending verification email')
 
+    const codeWithSpaces = code
+      .split('')
+      .filter(c => c)
+      .join('     ')
+
```

Comment:
whats this?

Reply:
removed it in a new commit. it was used for test